File: service/src/main/java/com/sisweb/api/entity/dto/UsuarioResetaSenhaDTO.java
package com.sisweb.api.entity.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import javax.validation.constraints.NotNull;

@Getter
@Setter
@ToString
public class UsuarioResetaSenhaDTO {

    @NotNull(message = "Campo obrigatório")
    private Long id;
    @NotNull(message = "Campo obrigatório")
    private Long timestampRecuperaSenha;
    private String nome;
    private String login;
    @NotNull(message = "Campo obrigatório")
    private String senha;
}

File: cliente/src/models/UsuarioResetaSenhaDTO.ts
export class UsuarioResetaSenhaDTO{
    id: Number;
    timestampRecuperaSenha: Number;
    nome: String;
    login: String;
    senha: String;

    setId(id: Number){
        this.id = id;
    }

    setTimestampRecuperaSenha( timestampRecuperaSenha: Number){
        this.timestampRecuperaSenha = timestampRecuperaSenha;
    }

    setNome(nome: String){
        this.nome = nome;
    }

    setLogin(login: String){
        this.login = login;
    }

    setSenha(senha: String){
        this.senha = senha;
    }
}